packages/schema/parse/src/abi/definitions.ts

```diff
@@ -477,11 +435,9 @@ export function createImportedEnumDefinition(args: {
   };
 }
 
+// TODO: We don't want this hard coded
```

Comment:
Yeah, it's alright because we need to hardcode it and we will be removing the concept of capabilities very soon. I have a few improvements in my mind already and Jordan also agrees with removing this.

Reply:
Agreed, once we treat all wrappers the same, and allow you to `invoke` any wrapper (including interface implementations) the same way we can remove this strange "capability" concept.

The `#use ...` syntax will also be removed from the schema.